plugins/resource-versions.yaml

```diff
@@ -3,7 +3,7 @@ kind: Plugin
 metadata:
   name: resource-versions
 spec:
-  version: v0.1.1
+  version: v0.1.2
```

Comment:
would it be worth adding some sort of check that the version here matches what's in the release asset?

cc @ahmetb @corneliusweig